static/docs/use-cases/data-registries.md

```diff
@@ -0,0 +1,195 @@
+  copies on other remotes). This simplifies data management and optimizes space
+  requirements.
+- Security: Registries can be setup to have read-only remote storage (e.g. an
+  HTTP location). Git versioning of [DVC-files](/doc/user-guide/dvc-file-format)
+  allows us to track and audit data changes.
+
+## Building registries
+
+Data registries can be created like any other <abbr>DVC repository</abbr> with
+`git init` and `dvc init`. A good way to organize them is with different
+directories, to group the data into separate uses, such as `images/`,
+`natural-language/`, etc. For example, our
+[dataset-registry](https://github.com/iterative/dataset-registry) uses a
+directory for each section in our website documentation, like `get-started/`,
+`use-cases/`, etc.
+
+Adding datasets to a registry can be as simple as placing the data file or
+directory in question inside the <abbr>workspace</abbr>, and telling DVC to
+track it, with `dvc add`. For example:
+
+```dvc
+$ mkdir -p music/Beatles
+$ cp ~/Downloads/millionsongsubset_full music/songs
+$ dvc add music/songs
+```
+
+> This example dataset actually exists. See
+> [MillionSongSubset](http://millionsongdataset.com/pages/getting-dataset/#subset).
+
+A regular Git workflow can be followed with the tiny
+[DVC-files](/doc/user-guide/dvc-file-format) that substitute the actual data
+(`music/songs.dvc` in this example). This enables team collaboration on data at
+the same level as with source code (commit history, branching, pull requests,
+reviews, etc.):
+
+```dvc
+$ git add music/songs.dvc music/.gitignore
+$ git commit -m "Track 1.8 GB 10,000 song dataset in music/"
+```
+
+> The actual data is stored in the project's <abbr>cache</abbr> and should be
+> [pushed](/doc/command-reference/push) to one or more
+> [remote storage](/doc/command-reference/remote) locations.
+
+## Using registries
+
+The main methods to consume <abbr>data artifacts</abbr> from a **data registry**
+are the `dvc import` and `dvc get` commands, as well as the `dvc.api.open()`
+function (Python).
+
+### Simple download (get)
+
+This is analogous to using direct download tools like
+[`wget`](https://www.gnu.org/software/wget/) (HTTP),
+[`aws s3 cp`](https://docs.aws.amazon.com/cli/latest/reference/s3/cp.html) (S3),
+etc. To get a dataset for example, we can run something like:
+
+```dvc
+$ dvc get git@git-server.url:path/to/repository.git \
+          path/to/dataset
+```
+
+This downloads `path/to/dataset` from the <abbr>project</abbr>'s
+[default remote](/doc/command-reference/remote/default) and places it in the
+current working directory (anywhere in the file system with user write access).
+
+> Note that this command (as well as `dvc import`) has a `--rev` option to
+> download specific versions of the data.
+
+### Import workflow
+
+`dvc import` uses the same syntax as `dvc get`:
+
+```dvc
+$ dvc import git@git-server.url:path/to/repository.git \
```

Comment:
if the intention is to keep it generic - let's use just <git-repo-url>?

Reply:
the whole notation `git@git-server.url:path/to/repository.git` feel too detailed. Also, you can do https:// which is simple, you don't need `git` protocol in a lot of cases

Reply:
OK. Going for `https://github.com/example/registry`.